documents/payment.md

```diff
@@ -863,6 +863,253 @@ Razorpay::Iin.fetch(tokenIin);
 }
 ```
 -------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------
+
+### Fetch a Payment (With Expanded EMI Details)
+
+```rb
+Razorpay::Payment.fetch("pay_XXXXXXXXXXXXXX").expendDetails({"expand[]": "emi"})
```

Comment:
Are we making two requests to the BE for this ?

Reply:
Yes it's like a chain request for better readablity.

We also used this in payment.
https://github.com/razorpay/razorpay-ruby/blob/master/documents/payment.md?plain=1#L250

Should I change it to a single request ?

Reply:
Request chaining does not seem optimal here. Two http requests are being made instead of a single request here ? Is there any reason why we have adopted this pattern in ruby sdk ?

Reply:
Made a single request instead of chaining